src/pages/trade-types/multiplier/_how-options-works.tsx

```diff
@@ -127,7 +127,7 @@ const HowOptionsWorks = () => {
                     >
                         {is_eu ? (
                             <StaticImage
-                                src="../../../images/common/trade-types/multiplier-market-eu.png"
```

Comment:
Hi @aswathy-deriv , can we delete the old images if it's not used anywhere?

Reply:
okay